src/library/transform/scan.cpp

```diff
@@ -1,147 +1,64 @@
-/* ************************************************************************
```

Comment:
We need the boilerplate back in this file.  You've done significant work on these files that you may want to consider adding Vratis copywrite below AMD.